src/ConstructiveSolidGeometry/LinePrimitives/Line.jl

```diff
@@ -1,61 +1,10 @@
-struct Line{T,TP,TL} <: AbstractLinePrimitive{T}
-    p1::TP
-    p2::TP
-    linetype::TL
-    function Line( ::Type{T},
-                   p1::PT,
-                   p2::PT,
-                   linetype::Union{Val{:inf}, Val{:ray}, Val{:seg}}
-                 ) where {T, PT <: Union{CartesianPoint{T}, PlanarPoint{T}}}
-        new{T,typeof(p1),typeof(linetype)}(p1, p2)
-    end
+struct Line{T} <: AbstractLinePrimitive{T}
+    origin::CartesianPoint{T}
+    normal::CartesianVector{T}
```

Comment:
Looking at the distance eq the `normal` is the vector that is parallel to the line. Shouldn't this called `direction` or line_vector instead of `normal`?

Reply:
I also though about this. I went for `normal` since I want it to be normalized to length 1.
But, I have no strong opinion on that one.

Reply:
I changed it to `direction`.